application/actions/meetings.go

```diff
@@ -301,3 +308,45 @@ func parseMeetingDates(input api.MeetingInput, modelMtg *models.Meeting) error {
 
 	return nil
 }
+
+// swagger:operation GET /events/{event_id} Events GetEvent
+//
+// gets one event/meeting with its participants
+//
+// ---
+// responses:
+//   '200':
+//     description: ab event/meeting
+//     schema:
+//       "$ref": "#/definitions/Meeting"
+func meetingsGet(c buffalo.Context) error {
+	cUser := models.CurrentUser(c)
+	tx := models.Tx(c)
+
+	id, err := getUUIDFromParam(c, "event_id")
+	if err != nil {
+		return reportError(c, err)
+	}
+
+	meeting := models.Meeting{}
+	if err = meeting.FindByUUID(tx, id.String()); err != nil {
+		appError := api.NewAppError(err, api.ErrorMeetingGet, api.CategoryNotFound)
+		if domain.IsOtherThanNoRows(err) {
+			appError.Category = api.CategoryInternal
+		}
+		return reportError(c, appError)
+	}
+
+	canViewParticipants, err := cUser.CanCreateMeetingInvite(tx, meeting)
```

Comment:
Should this be a separate authz function like `CanViewParticipants`?

Reply:
Or maybe a more broad `CanManageMeeting` that both can use?

Reply:
:+1:

Reply:
I think we've talked in the past about in some events we'd want to let participants invite other participants so it's not only controlled by the host. It was something about visibility of the event and whether an invitation is required to join. So basing whether they can view participants on the same check could become problematic later.